examples/run_workflows_ReactAims.py

```diff
@@ -20,7 +20,7 @@ def test_run_workflows_ReactAims():
     os.chdir("data/react")
 
     '''Determine calculation input settings'''
-    params = {"xc":"pbe"}
+    params = {"xc":"libxc MGGA_X_MBEEF+GGA_C_PBE_SOL", "compute_forces": '.true.'}
```

Comment:
Is there a reason this has been changed to the more complex libxc option? Is this to make sure the testing works correctly? An explanation would be good.

Reply:
This change is because the change of the `if ` statement in` run/workflow/react.py` line 593. If compute_forces is not set, line 594 will not be covered by the CI tests.

Reply:
I agree that libxc should be tested for coverage, but please replace it with e.g. PBE libxc implementation

Reply:
Resolved as revert to the original code which always set override_libxc to True.

Reply:
I'm not a great fan of this - we shouldn't default warning flags off as then people can make mistakes with code usage (@PavelStishenko and Lana hit this this issue only last week). Is there a better default we can find?